src/main/resources/templates/kotlin-lang/kotlinClassGraphqlResponseProjection.ftl

```diff
@@ -0,0 +1,94 @@
+<#if package?has_content>
+package ${package}
+
+</#if>
+import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResponseField
+import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResponseProjection
+<#if equalsAndHashCode>
+import java.util.Objects
+</#if>
+
+<#if javaDoc?has_content>
+/**
+<#list javaDoc as javaDocLine>
+ * ${javaDocLine}
+</#list>
+ */
+</#if>
+<#if generatedInfo.getGeneratedType()?has_content>
+@${generatedInfo.getGeneratedType()}(
```

Comment:
We need to be careful with the default value of this for Kotlin generation.
I'm not sure there is a Kotlin equivalent to `javax.annotations.Generated`, so I guess by default we should put nothing, because Kotlin is not necessarily intended for the JVM.

Reply:
The premise of our library is the JVM language, otherwise it does not match the library name.
If we think that way, we won't be able to fully support any other language. Because many other JVM languages are actually cross platform, such as Scala and kotlin

Reply:
The name of the library is actually only Java, not the JVM. Bringing in more languages is already out of the scope of the initial library, so I'm not sure we can reason this way.

The question is whether the generated code actually needs library classes that are JVM only. While I think this is the case at the moment, I'm not sure it needs to be this way forever.
But I agree that if this is changed, the change has nothing to do with this particular PR, so it could wait.

Reply:
I agree that clients may be used across platforms.